application/web/js/namespace.js

```diff
@@ -6,5 +6,11 @@
         throw new Error('PacificaSearch is already defined, did you include this file twice?');
     }
 
-    window.PacificaSearch = {};
+    window.PacificaSearch = {
+        /**
+         * How many items to load initially into each faceted search type, and how many to load on each click of the
+         * "more" link.
+         */
+        FACETED_SEARCH_ITEMS_PER_PAGE : 3
```

Comment:
Double-check that this is being used. Aren't we just passing the page number and not the size? Either way this is redundant with the PHP constant, so one of the two should probably be removed